internal/native/chan.go

```diff
@@ -36,11 +36,11 @@ func (n *Native) handleVideoStateChan() {
 func (n *Native) handleLogChan() {
 	for {
 		entry := <-logChan
-		l := n.l.With().
+		l := GetNativeLogger().
```

Comment:
This is much closer to the final changes... mostly swapping member variables with calls to a function